git-crypt/PKGBUILD

```diff
@@ -0,0 +1,20 @@
+pkgname="git-crypt"
```

Comment:
Add header

Reply:
Amended. I hope the header is just the note about maintainer, which I put myself in (I'm willing to take care of the package in the future).